libs/liblognorm/Makefile

```
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2010-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=liblognorm
+PKG_VERSION:=2.0.6
+PKG_RELEASE:=2
```

No reason to bump `PKG_RELEASE` to adding new package

```suggestion
PKG_RELEASE:=1
```